packages/clerk-js/src/ui/elements/Action/ActionCard.tsx

```diff
@@ -2,17 +2,20 @@ import type { ComponentProps } from 'react';
 
 import { Col } from '../../customizables';
 
-type ActionCardProps = ComponentProps<typeof Col>;
+type ActionCardProps = ComponentProps<typeof Col> & {
+  isDestructive?: boolean;
```

Comment:
For consistency, we should go with:
```suggestion
  variant?: 'neutral' | 'destructive';
```

Reply:
Updated.